frontstage/models.py

```diff
@@ -133,17 +69,12 @@ def validate_password(form, field):
 
 
 class LoginForm(FlaskForm):
-    """Login form."""
-
     username = StringField('Email Address', [InputRequired("Email Address is required"),
```

Comment:
Is there reasoning behind removing all the docstrings?

Reply:
We don't use them consistently at the moment.

And also a docstring that just repeats the name of the class doesn't seem that useful

Reply:
Wouldn't it be more useful to update the docstring rather than to delete it?

Reply:
I don't see the point in writing up one doctring for this form right now. 
We should probably agree as a team how we're going to document our code across RAS?

Reply:
I thought as a dev team we decided on PEP 8 if so docstrings are part of PEP 8 are they not?

Reply:
Yep I believe so https://www.python.org/dev/peps/pep-0257/
I feel like adding these in is a much wider task than is covered in this PR however seeing as there's so much to be documented

Reply:
I think this needs to be brought to the attention of Dave.

Reply:
Or add a todo comment to make sure they get done later.